zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/api/scheduling/SimpleProcessingScheduleService.java

```diff
@@ -35,6 +35,14 @@ public interface SimpleProcessingScheduleService {
    */
   ScheduledTask runAt(long timestamp, Task task);
 
+  /**
+   * Schedules the task to run at or after the given timestamp.
+   *
+   * @implNote Can be silently ignored if the scheduling service is not ready.
+   * @return A representation of the scheduled task.
+   */
+  ScheduledTask runAt(long timestamp, Runnable task);
```

Comment:
I think you need to implement this method in `TestScheduleServiceActorDecorator` to overcome the compilation error.

Reply:
Ah, I missed that, thanks @mustafadagher!